client/src/pages/Logout.js

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from "react";
+import styled from "styled-components/macro";
+import Page from "../common/Page";
+import { AuthContext } from "../contexts/AuthContext";
+
+export default function Logout() {
+  const { user, logout } = useContext(AuthContext);
```

Comment:
Code-Style

```suggestion
  const { user, logout } = useAuth();
```

with

```js
  function useAuth() {
    return useContext(AuthContext);
  }

  function useUser() {
    return useAuth().user;
  }
```